app/components/Form/ProjectRfpForm.tsx

```diff
@@ -0,0 +1,130 @@
+import { Button } from "@button-inc/bcgov-theme";
+import FormBase from "components/Form/FormBase";
+import SelectRfpWidget from "components/Form/SelectRfpWidget";
+import projectSchema from "data/jsonSchemaForm/projectSchema";
+import { JSONSchema7, JSONSchema7TypeName } from "json-schema";
+import EmptyObjectFieldTemplate from "lib/theme/EmptyObjectFieldTemplate";
+import { useCreateProjectMutation } from "mutations/Project/createProject";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { graphql, useFragment } from "react-relay";
+import {
+  getExternalProjectRevisionFormPageRoute,
+  getProjectRevisionFormPageRoute,
+} from "routes/pageRoutes";
+
+interface Props {
+  query: any;
```

Comment:
Let's replace this `any` type. Maybe something like this?
```suggestion
  query: ProjectRfpForm_query$key;
```